### PR TITLE
Add --show-events flag to test command

### PR DESCRIPTION
**Pull request by xitter**
Issue https://github.com/trufflesuite/truffle/issues/186

Referenced PR on archived repo https://github.com/trufflesuite/truffle-core/pull/6

`truffle test --show-events`

### CODE REVIEW

**Comment by gnidan** [comment]
Hey @xitter, thanks for this!

Could you also add this as an option to `packages/truffle-core/lib/commands/test.js`, in the `builder` property for the command? See this example for the [compile](https://github.com/trufflesuite/truffle/blob/develop/packages/truffle-core/lib/commands/compile.js#L5-L8) command.

Just so it's slightly more explicit.

File: packages/truffle-core/lib/testing/testrunner.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ TestRunner.prototype.startTest = function(mocha, callback) {
 TestRunner.prototype.endTest = function(mocha, callback) {
   var self = this;
 
-  if (mocha.currentTest.state != "failed") {
+  if (mocha.currentTest.state != "failed" && !self.config["show-events"]) {
```
Comment (gnidan) [inline packages/truffle-core/lib/testing/testrunner.js]: We also probably want a comment here because it's not entirely clear.

**Comment by xitter** [comment]
@gnidan Added PR https://github.com/trufflesuite/truffle/pull/1238 for adding this as an option in truffle-core.

**Comment by gnidan** [comment]
Great, thanks! Merging.

**Pull request merged**